packages/wallet-backend-update/src/rapyd/rapyd-client.ts

```diff
@@ -0,0 +1,248 @@
+import axios, { AxiosError, AxiosResponse } from 'axios'
+import crypto from 'crypto-js'
+import { Logger } from 'winston'
+import { Env } from '@/config/env'
+
+interface IRapydClient {
+  createWallet(
+    wallet: RapydWallet
+  ): Promise<AxiosResponse<RapydResponse<RapydWallet>>>
```

Comment:
Can `AxiosResponse` be dropped? In the `request` function we directly return the data.